### PR TITLE
 Section where I will list my Portfolio Projects

### PR DESCRIPTION
**Pull request by Kanyorok**
This is the second milestone in building my portfolio website where I created the mobile website section where I will list your portfolio projects.

### CODE REVIEW

File: index.html
Comment (aamirkhan2478) [inline index.html:1]: - [ ] Please fix your full mobile version there are lots of spacing problems kindly fix it and stick to the Figma design :+1: check below :point_down: my video :slightly_smiling_face: 

**Video**

[screen-capture (11).webm](https://user-images.githubusercontent.com/45226848/235296527-2fed6ba8-1988-4b52-859a-d6dca3075446.webm)

File: index.html [hunk 1]
```diff
@@ -60,6 +71,242 @@ <h2 class="subtitle">I'm a Software Developer</h2>
...
+            </li>
+            <li>
+              <a href="#">BOOTSTRAP</a>
+            </li>
+            <li>
+              <a href="#">RUBY</a>
+            </li>
+          </ul>
+          <button type="button" >See project</button>
+        </article>
```
Comment (aamirkhan2478) [inline index.html:125]: - [ ] Please use all button effects I notice your pressed effect is not working so kindly fix this issue :+1:  check below :point_down: the pressed button effect :slightly_smiling_face: 

Screenshot from 2023-04-29 14-53-54

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @Kanyorok , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linters passed

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @Kanyorok, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Removed all the extra space ✔️ 
- Added active button interaction ✔️ 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
* Noting to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**